src/main/java/uk/gov/hmcts/darts/common/helper/MediaLinkedCaseHelper.java

```diff
@@ -11,18 +12,30 @@
 
 @Component
 @RequiredArgsConstructor
+@Slf4j
 public class MediaLinkedCaseHelper {
 
     private final MediaLinkedCaseRepository mediaLinkedCaseRepository;
 
     public void addCase(MediaEntity mediaEntity, CourtCaseEntity courtCase) {
         List<MediaLinkedCaseEntity> mediaLinkedCaseEntities = mediaLinkedCaseRepository.findByMedia(mediaEntity);
         List<CourtCaseEntity> linkedCases = mediaLinkedCaseEntities.stream().map(MediaLinkedCaseEntity::getCourtCase).toList();
+        log.debug("Handling med_id {} and cas_id {}. Currently the media has the following cas_ids linked: {}",
```

Comment:
Can we put this inside a if (log.isDebugEnabled()) statement just so loop over the linkedCase array when we do not need to

Reply:
Have now added this as a guard.